<?xml version="1.0"?>
<!-- Launches: ardrone_autonomy driver;-->
<launch>
      	<arg name="drone_id_namespace" default="drone0" />
	<group ns="$(arg drone_id_namespace)">
            <!-- Ardrone driver node -->
	    <node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" clear_params="true" args="-ip $(arg drone_ip)">
               <param name="navdata_demo" value="0" />
               <param name="realtime_navdata" value="true"/>
	    </node>
        </group>
</launch>
